includes/definitions/dasan-nos.yaml

```diff
@@ -0,0 +1,10 @@
+os: dasan-nos
+text: 'Dasan NOS'
+type: network
+icon: dasan
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    method: sysObjectId
+    sysObjectId:
+        - .1.3.6.1.4.1.6296.1.2.5.12
```

Comment:
Perhaps something like this:
```
discovery:
    - sysObjectId: .1.3.6.1.4.1.6296.
    - sysDescr: 'Description'
    - { sysObjectId: .1.3.6.1.4.1.6296., sysDescr: 'Description' }
``` 

The first two lines would match either sysObjectId or sysDescr.
The third line would require both to match.